mon-pix/app/components/certification-joiner.js

```diff
@@ -60,17 +58,27 @@ export default class CertificationJoiner extends Component {
     this.showCongratulationsBanner = false;
   }
 
+  _isAValidSessionId(value) {
+    const isANumberRegex = new RegExp(this.sessionPattern);
+    return isANumberRegex.test(value);
```

Comment:
on peut essayer de le one-liner ?
`return new RegExp(this.sessionPattern).test(value);`